unity-renderer/Assets/Scripts/MainScripts/DCL/WorldRuntime/SceneController.cs

```diff
@@ -28,24 +31,33 @@ public void Initialize()
             lastSortFrame = 0;
             enabled = true;
 
+            loadingFeedbackController = new LoadingFeedbackController();
+
             DataStore.i.debugConfig.isDebugMode.OnChange += OnDebugModeSet;
 
             if (deferredMessagesDecoding) // We should be able to delete this code
-                deferredDecodingCoroutine = CoroutineStarter.Start(DeferredDecoding()); //
+                deferredDecodingCoroutine = CoroutineStarter.Start(DeferredDecoding());
 
             DCLCharacterController.OnCharacterMoved += SetPositionDirty;
 
             CommonScriptableObjects.sceneID.OnChange += OnCurrentSceneIdChange;
 
-            //TODO(Brian): Move those subscriptions elsewhere.
-            PoolManager.i.OnGet -= Environment.i.platform.physicsSyncController.MarkDirty;
-            PoolManager.i.OnGet += Environment.i.platform.physicsSyncController.MarkDirty;
-
-            PoolManager.i.OnGet -= Environment.i.platform.cullingController.objectsTracker.MarkDirty;
-            PoolManager.i.OnGet += Environment.i.platform.cullingController.objectsTracker.MarkDirty;
+            // TODO(Brian): Move this later to Main.cs
+            if ( !EnvironmentSettings.RUNNING_TESTS )
+            {
+                if (prewarmSceneMessagesPool)
+                {
+                    for (int i = 0; i < 100000; i++)
```

Comment:
It would be nice to move the `100000` value to a constant with a descriptive name.